front/lib/auth.ts

```diff
@@ -797,6 +801,11 @@ export class Authenticator {
     return this._groups.map((g) => g.toJSON());
   }
 
+  /**
+   * Checks if the authenticator has the specified permission across all ACLs.
```

Comment:
Would be nice to explicit the role of each arguments here:

acls: the ACL of the resource being checked
permissions: the permission being checked

(If I'm not being mistaken)